components/layouts/blog/blog-filter.config.json

```diff
@@ -0,0 +1,3 @@
+{
+	"status": "wip"
```

Comment:
Don't hate me for this, but could we refactor this pattern a little? Rather than have a "blog filter bar" pattern in the blog patterns, let's make two specific changes.

1. Move it to /template/filter-bar - reasoning being that /layouts are more for content. This is a template level component-type item that isn't meant to be user editable or anything.
2. Make it location agnostic. While we have a blog filter bar, it exists in at least two other contexts now (client stories and jobs). So just have a "filter-bar" pattern, and demonstrate it with and without a sort option, since it's otherwise the same across usages.